tests/examples/opinions/united_states/bap1_example.html

```diff
@@ -0,0 +1,520 @@
alize.css":1,"sites\/all\/themes\/adaptive\/css\/adaptive.hacks.css":1,"sites\/all\/themes\/adaptive\/css\/adaptive.styles.css":1,"sites\/all\/themes\/appeals\/css\/appeals.normalize.css":1,"sites\/all\/themes\/appeals\/css\/appeals.hacks.css":1,"sites\/all\/themes\/appeals\/css\/appeals.styles.css":1,"sites\/all\/themes\/cstbase\/css\/cstbase.no-query.css":1,"sites\/all\/themes\/adaptive\/css\/adaptive.no-query.css":1,"sites\/all\/themes\/appeals\/css\/appeals.no-query.css":1,"all:0":1,"public:\/\/css_injector\/css_injector_5.css":1,"public:\/\/css_injector\/css_injector_7.css":1}},"jQueryUiFilter":{"disabled":0,"accordionHeaderTag":"h3","accordionOptions":{"active":0,"animated":"slide","autoHeight":"true","clearStyle":"false","collapsible":"false","event":"click","scrollTo":0,"history":"false"},"tabsOptions":{"collapsible":"false","event":"click","fx":0,"paging":{"back":"\u0026laquo; Previous","next":"Next \u0026raquo;"},"selected":"","scrollTo":0,"history":"false"},"tabsHeaderTag":"h3"},"custom_search":{"form_target":"_self","solr":0},"views":{"ajax_path":"\/views\/ajax","ajaxViews":{"views_dom_id:e74aee4e94ba5c3c5fbc3ef5cba424ed":{"view_name":"opn_files_view","view_display_id":"bapopn_search_results","view_args":"","view_path":"bapopn","view_base_path":"bapopn","view_dom_id":"e74aee4e94ba5c3c5fbc3ef5cba424ed","pager_element":0}}},"urlIsAjaxTrusted":{"\/views\/ajax":true,"\/bapopn":true},"responsive_menus":[{"selectors":"#main-menu, #block-superfish-1, .l-region--header .menu-name-main-menu","container":"body","trigger_txt":"\u0026#8801; MENU","close_txt":"X CLOSE","close_size":"1em","position":"left","media_size":"960","show_children":"1","expand_children":"1","expand_txt":"\u25bc","contract_txt":"\u25b2","remove_attrs":"1","responsive_menus_style":"mean_menu"}],"us_courts_extlink":{"extlink_default_text":"You are now leaving the website of the First Circuit. The link below contains information created and maintained by other public and private organizations.\n\nThis link is provided for the user\u0027s convenience. The First Circuit does not control or guarantee the accuracy, relevance, timeliness, or completeness of this outside information; nor does it control or guarantee the ongoing availability, maintenance, or security of this Internet site. Further, the inclusion of links is not intended to reflect their importance or to endorse any views expressed, or products or services offered, on these outside sites, or the organizations sponsoring the sites."},"extlink":{"extTarget":0,"extClass":0,"extLabel":"(link is external)","extImgClass":0,"extSubdomains":1,"extExclude":"(.uscourts.gov)|(-admin.jdc.ao.dcn)|(-dev.jdc.ao.dcn)","extInclude":"","extCssExclude":".main-slider","extCssExplicit":"","extAlert":0,"extAlertText":"This link will take you to an external web site. We are not responsible for their content.","mailtoClass":0,"mailtoLabel":"(link sends e-mail)"},"password":{"strengthTitle":"Password compliance:"},"type":"setting"});</script>
```

Comment:
Unless it is relevant to the test - you should strip out the junk... 

the things that aren't needed just to keep file sizes down and tests fast